src/lib/Navbar.svelte

```diff
@@ -172,6 +172,16 @@
 		}
 	}
 
+	@media print {
+		header {
+			position: relative;
```

Comment:
Why relative instead of static? Since we're not offsetting its position

Reply:
Oh yeah I forgot about static. I'm just used to relative I guess. I can do static if you prefer.

Reply:
No relative is fine. I was just trying to understand if you picked relative for a specific reason that I didn't understand.

Reply:
Not really. The main difference between static and relative here is that relative creates a stacking context but I don't think the presence of a stacking context would change anything with the current CSS